floors/enforce.go

```diff
@@ -154,6 +155,14 @@ func enforceFloorToBids(bidRequestWrapper *openrtb_ext.RequestWrapper, seatBids
 	return seatBids, errs, rejectedBids
 }
 
+// isBidPriceLowerThanBidfloor checks for bid price with bid floor with given precision
+func isBidPriceLowerThanBidfloor(bidFloor float64, bidPrice float64) bool {
+	if bidPrice < bidFloor && math.Abs(bidFloor-bidPrice) > floorPrecision {
```

Comment:
Would this not be the same as `(bidPrice + floorPrecision) < bidFloor`

Reply:
corrected